test/ActivityRepository-test.js

```diff
@@ -0,0 +1,99 @@
+import { expect } from 'chai';
+import Activity from '../src/Activity';
+import ActivityRepository from '../src/ActivityRepository';
+import UserRepository from '../src/UserRepository';
+
+describe('ActivityRepository', () => {
+  let activityRepo, data, userData, userRepo
+
+  beforeEach(() => {
+    data = [
+        {"userID": 1, "date": '2019/06/15', "numSteps": 3700, "minutesActive": 10, 'flightsOfStairs': 20},
+        {"userID": 1, "date": '2019/06/16', "numSteps": 4294, "minutesActive": 41, 'flightsOfStairs': 14},
+        {"userID": 1, "date": '2019/06/17', "numSteps": 7402, "minutesActive": 20, 'flightsOfStairs': 42},
+        {"userID": 1, "date": '2019/06/18', "numSteps": 10000, "minutesActive": 10, 'flightsOfStairs': 31},
+        {"userID": 1, "date": '2019/06/19', "numSteps": 11374, "minutesActive": 97, 'flightsOfStairs': 63},
+        {"userID": 1, "date": '2019/06/20', "numSteps": 2643, "minutesActive": 21, 'flightsOfStairs': 23},
+        {"userID": 1, "date": '2019/06/21', "numSteps": 6389, "minutesActive": 40, 'flightsOfStairs': 43},
+        {"userID": 1, "date": '2019/06/22', "numSteps": 11652, "minutesActive": 76, 'flightsOfStairs': 31},
+        {"userID": 2, "date": '2019/06/15', "numSteps": 4230, "minutesActive": 82, 'flightsOfStairs': 12},
+        {"userID": 2, "date": '2019/06/16', "numSteps": 9842, "minutesActive": 50, 'flightsOfStairs': 31},
+        {"userID": 2, "date": '2019/06/17', "numSteps": 5023, "minutesActive": 40, 'flightsOfStairs': 41}
+      ]
+      const userData = [{
```

Comment:
Hey!  Not sure if this is supposed to be a const, it has the same name as the let before the beforeEach statement.